llvm/include/llvm/ADT/APFloat.h

```diff
@@ -1397,6 +1397,8 @@ inline APFloat minnum(const APFloat &A, const APFloat &B) {
     return B;
   if (B.isNaN())
     return A;
+  if (A.isZero() && B.isZero() && (A.isNegative() != B.isNegative()))
```

Comment:
Will using:
```
if (A == B) return A.isNegative() ? A : B;
```
be a bit more efficient?

Reply:
Not sure, I just copied this from the minimum/maximum implementations. Probably should follow the same pattern and change all of them together if it makes sense

Reply:
IIUC this function now implements IEEE 2019 minimumNumber which is a refinement of IEEE 2008 minNum. Should update the doxygen comment accordingly.